### PR TITLE
Fix scrolling issue in section edit dialog

### PR DESCRIPTION
**Pull request by epeach**
This Axosoft Item: https://codeorg.axosoft.com/viewitem?id=1425&type=features&force_use_number=true

Basically, there were two nested scrollbars on the sections edit dialog. I removed the outer one and updated the inner one to scroll the contents of the dialog leaving the title and footer buttons fixed.

Before:
before

After:
after

### CODE REVIEW

File: apps/src/templates/BaseDialog.jsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ var BaseDialog = React.createClass({
     }
     if (this.props.fullHeight) {
```
Comment (epeach) [inline apps/src/templates/BaseDialog.jsx:82]: This condition was added recently for a feature that got re-vamped to not use it. No other use cases in the code, so adjusting this value should have no ill effect.

File: apps/src/templates/BaseDialog.jsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ var BaseDialog = React.createClass({
         height: this.props.fixedHeight,
         maxHeight: !this.props.fixedHeight && '80vh',
         overflowX: 'hidden',
-        overflowY: this.props.fixedHeight ? 'hidden' : 'auto',
+        overflowY: (this.props.fixedHeight || this.props.fullHeight) ? 'hidden' : 'auto',
```
Comment (epeach) [inline apps/src/templates/BaseDialog.jsx:101]: FullHeight is logically equivalent to fixedHeight in this case (both are component props that indicate a specific height). Including fullHeight in this check to allow scrolling for all base dialogs with a specified height.

File: apps/src/templates/BaseDialog.jsx (hunk 1 above)
Comment (caleybrock) [reply apps/src/templates/BaseDialog.jsx:82]: If it's not used should we follow up by removing the code?

Comment (caleybrock) [reply apps/src/templates/BaseDialog.jsx:82]: Sorry, I misunderstood this. This sounds good.

**Pull request merged**